data-prepper-plugins/parquet-codecs/build.gradle

```diff
@@ -0,0 +1,23 @@
+plugins {
```

Comment:
Please remove the `plugins` and `repositories` sections. They are inherited from the root project.

Reply:
done